app/main.py

```diff
@@ -1,7 +1,34 @@
+import os
+
+import dotenv
```

Comment:
Ensure that the `dotenv` package is installed in your environment, as it is required to load environment variables from a `.env` file.